test/gen_casadi_test.py

```diff
@@ -2,39 +2,39 @@
 """
 Modelica parse Tree to AST tree.
 """
-from __future__ import print_function, absolute_import, division, unicode_literals
+from __future__ import absolute_import, division, print_function, unicode_literals
 
-import os
 import glob
+import os
+import re
 import unittest
-import itertools
 
 import casadi as ca
+
 import numpy as np
 
 import pymoca.backends.casadi.generator as gen_casadi
+from pymoca import parser
 from pymoca.backends.casadi.alias_relation import AliasRelation
+from pymoca.backends.casadi.api import CachedModel, transfer_model
 from pymoca.backends.casadi.model import (
     CASADI_ATTRIBUTES,
+    DelayArgument,
     Model,
-    Variable,
     StringVariable,
-    DelayArgument,
+    Variable,
 )
-from pymoca.backends.casadi.api import transfer_model, CachedModel
-from pymoca import parser, ast
 
 
 MODEL_DIR = os.path.join(os.path.dirname(os.path.realpath(__file__)), "models")
 
 
-# noinspection PyPep8Naming,PyUnresolvedReferences
 class GenCasadiTest(unittest.TestCase):
     def assert_model_equivalent(self, A, B):
         def sstr(a):
-            return set([str(e) for e in a])
+            return {str(e) for e in a}
 
-        for l in [
+        for vgroup in [
```

Comment:
Yep, choose `vgroup` exactly because of that "used as such elsewhere". I don't disagree it's a bad name though, can make a commit on top renaming it to something that makes more sense?